plotter/histogram.go

```diff
@@ -106,6 +115,16 @@ func (h *Histogram) DataRange() (xmin, xmax, ymin, ymax float64) {
 		if bin.Weight > ymax {
 			ymax = bin.Weight
 		}
+		if bin.Weight < ymin {
+			ymin = bin.Weight
+		}
+		if 0 < bin.Weight && bin.Weight < ylow {
```

Comment:
This reads as a range check; bin.Weight in (0, ylow). But I don't think this is what you are doing; you are checking whether bin.Weight is non-zero and only using it then. Is this correct? If so, please rephrase this as `if bin.Weight != 0 && bin.Weight < ylow {`. If it's a check that bin.Weight is positive non-zero, then replace the `!=` in that with a `>`.